.devcontainer/analyst/Dockerfile

```diff
@@ -1,2 +1,3 @@
 FROM ghcr.io/nasa/isaac_analyst_notebook
 ENV DEBIAN_FRONTEND=dialog
+RUN pip install torch==2.2.1 torchvision==0.17.1 torchaudio==2.2.1 --index-url https://download.pytorch.org/whl/cu121
```

Comment:
Do we need to commit to these exact versions? What is this url?

Reply:
the link is from the pytorch install instructions here: https://pytorch.org/

Reply:
the exact versions might be remaining from when we had to support multiple ubuntu versions